_posts/2025-04-02-indico-workshop-2025.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Indico workshop returns - we need your help!"
+date: 2025-04-02 17:00
+author: "Indico Team"
+categories: indico workshop
+---
+
+![](/assets/2025-04-02-indico-workshop-2025/workshop_2025.png){: width="60%" style="box-shadow: none" }
+
+
+On the **6th and 7th of May 2025**, the [Indico workshop](https://indico.cern.ch/e/indico-workshop/2025) will return, where developers, administrators and representatives of organizations in the Indico community will meet in Geneva, at **CERN** and the **United Nations**, to discuss the latest developments and future plans for the Project.
+
+The workshop will be held in a hybrid format, allowing participants to join either in person or remotely. This is a great opportunity for those who are unable to travel to Geneva to still participate in the discussions and share their insights.
```

Comment:
I would not announce remote connections here TBH - we want to encourage people to attend in person. like the last time, we'll provide the zoom option a few days before.